support-frontend/assets/pages/paper-subscription-checkout/helpers/deliveryDays.js

```diff
@@ -0,0 +1,89 @@
+// @flow
+import {
+  Saturday,
+  SaturdayPlus,
+  Sunday,
+  SundayPlus,
+  Weekend,
+  WeekendPlus,
+  Sixday,
+  SixdayPlus,
+  Everyday,
+  EverydayPlus,
+  type PaperProductOptions,
+} from 'helpers/productPrice/productOptions';
+
+type Day = 0 | 1 | 2 | 3 | 4 | 5 | 6;
+const milsInADay = 1000 * 60 * 60 * 24;
+
+const maxWeeks = 4;
```

Comment:
I find maxWeeks a bit of a vague name - is it the furthest date in the future that a user can choose to start their subscription?

Reply:
yes!